eth2near-relay/index.js

```diff
@@ -21,7 +21,7 @@ function execute(command, _callback) {
 }
 
 function web3BlockToRlp(blockData) {
-  blockData.difficulty = parseInt(blockData.difficulty, 10)
+  blockData.difficulty = parseInt(blockData.difficulty || '0', 10)
```

Comment:
Please add the comment that `difficulty` should be always present on PoW Ethereum networks.